Assets/MRTK/SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -868,14 +868,15 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
         /// </summary>
         public void OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            bool isNear = eventData.Pointer as IMixedRealityNearPointer != null;
             oldIsTargetPositionLockedOnFocusLock = eventData.Pointer.IsTargetPositionLockedOnFocusLock;
             if (!(eventData.Pointer is IMixedRealityNearPointer) && eventData.Pointer.Controller.IsRotationAvailable)
```

Comment:
This looks like the same check that's now stored in `isNear`, so we could reuse the new bool here too